example/webpack.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     loaders: [
       {
         test: /\.js$/,
-        loader: 'babel?presets=react!' + require.resolve('../lib/webpackLoader') + '?LayoutConstants=' + require.resolve('./LayoutConstants'),
```

Comment:
Removed this because custom class name strategy doesn't currently work with the loader.